store/src/main/java/com/nytimes/android/external/store/base/impl/BarCode.java

```diff
@@ -1,5 +1,7 @@
 package com.nytimes.android.external.store.base.impl;
 
+import com.nytimes.android.external.cache.Preconditions;
```

Comment:
the logic here was that cache was extracted from guava, anything that had to come with it will live in Cache package so that some user can include only Cache